py/torch_tensorrt/dynamo/conversion/impl/embedding.py

```diff
@@ -94,7 +94,10 @@ def embedding_bag_with_traversable_offsets(
         # however, pytorch doc says if `include_last_offset` is True, the size of offsets
         # is equal to the number of bags + 1. The last element is the size of the input,
         # or the ending index position of the last bag (sequence).
-        offsets.itemset(-1, len_embed)
+        offsets_shape = offsets.shape
+        offsets = offsets.flatten()
+        offsets[-1] = len_embed
+        offsets.reshape(offsets_shape)
```

Comment:
According to the error: 
```
AttributeError: `itemset` was removed from the ndarray class in NumPy 2.0. Use `arr[index] = value` instead.
```
maybe you can just use `offsets[-1] = len_embed`?

Reply:
@zewenli98  for the multi dimensional arry, offset[-1] is actually updating the last dimension as the value
what we want is only to update the last element as the value
```
>>> l = torch.rand(2,3)
>>> l
tensor([[0.0770, 0.6687, 0.1859],
        [0.5044, 0.9199, 0.2451]])
>>> l[-1]=9.99
>>> l
tensor([[0.0770, 0.6687, 0.1859],
        [9.9900, 9.9900, 9.9900]])
```

Reply:
This usage is correct for a general case, but I think in this case the `offsets` is 1d, per https://pytorch.org/docs/stable/generated/torch.nn.EmbeddingBag.html

Reply:
```
A few notes about input and offsets:

input and offsets have to be of the same type, either int or long

If input is 2D of shape (B, N), it will be treated as B bags (sequences) each of fixed length N, and this will return B values aggregated in a way depending on the mode. offsets is ignored and required to be None in this case.

If input is 1D of shape (N), it will be treated as a concatenation of multiple bags (sequences). offsets is required to be a 1D tensor containing the starting index positions of each bag in input. Therefore, for offsets of shape (B), input will be viewed as having B bags. Empty bags (i.e., having 0-length) will have returned vectors filled by zeros.
```